.tmuxinator.yml

```diff
@@ -49,9 +49,8 @@ windows:
       # 'before' represents legacy functionality and will be deprecated in a future release, in favour of 'after'
       # synchronize: after
       panes:
-        - sleep 1 && $cur__target_dir/default/bin/http_server.exe 0
-        - sleep 1 && $cur__target_dir/default/bin/http_server.exe 1
-        - sleep 1 && $cur__target_dir/default/bin/http_server.exe 2
-        - sleep 1 && $cur__target_dir/default/bin/http_server.exe 3
-        - $cur__target_dir/default/bin/sidecli.exe make-credentials && sleep 3 && $cur__target_dir/default/bin/sidecli.exe inject-genesis
+        - sleep 1 && esy x http_server 0
+        - sleep 2 && esy x http_server 1
+        - sleep 3 && esy x http_server 2
+        - sleep 4 && esy x sidecli inject-genesis
```

Comment:
Does this no longer cause issues with esy?

Reply:
Actually it seems to work fine. Does that mean [this](https://github.com/marigold-dev/sidechain/blame/b31a21169c9efad588487268bf03bd4cd6b0941b/README.md#L170) should be changed, or does that still not work?

Reply:
Yes, that entire section about esy not managing lock files can go away